src/coreComponents/physicsSolvers/fluidFlow/wells/CompositionalMultiphaseWell.cpp

```diff
@@ -126,15 +126,18 @@ void CompositionalMultiphaseWell::postInputInitialization()
   WellSolverBase::postInputInitialization();
 
   GEOS_ERROR_IF_GT_MSG( m_maxCompFracChange, 1.0,
-                        getWrapperDataContext( viewKeyStruct::maxCompFracChangeString() ) <<
-                        ": The maximum absolute change in component fraction must smaller or equal to 1.0" );
+                        viewKeyStruct::maxCompFracChangeString() <<
+                        "The maximum absolute change in component fraction must smaller or equal to 1.0",
```

Comment:
```suggestion
                        "The maximum absolute change in component fraction must smaller or equal to 1.0",
```
and same below below